test/e2e/utils/recordedit-helpers.js

```diff
@@ -321,7 +347,51 @@ exports.testPresentationAndBasicValidation = function(tableParams, isEditMode) {
                     });
                 });
             }
+            
+            if (jsonCols.length > 0) {
+                describe("JSON fields, ", function () {
+                    it("should show textarea input for JSON datatype and then set the value", function() {
+                        jsonCols.forEach(function(c) {
+                        chaisePage.recordEditPage.getTextAreaForAcolumn(c.name, recordIndex).then(function(jsonTxtArea) {
+                            if (jsonTxtArea) {
+                                expect(true).toBeDefined();
+                                jsonTxtArea.column = c;
+                                
+                                JSONDataTypeFields.push(jsonTxtArea);
+                                var value = getRecordValue(c.name);
 
+                                if (value != undefined) {
```

Comment:
@jrchudy This is fine. The `value` is null if we don't provide any current value to the `testPresentationAndBasicValidation` function (when we don't want to test the current value of the inputs). For example in case of create, we don't need to test the current value.